apps/rush-lib/src/api/BaseFlagFile.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { FileSystem, JsonFile, JsonObject } from '@rushstack/node-core-library';
+
+/**
+ * A helper class for managing last-install flags, which are persistent and
```

Comment:
This sounds overly specific. Is this comment outdated?

Reply:
Oops, copy-paste :)